SAMPLES.md

```diff
@@ -0,0 +1,104 @@
+# Backchannel integration
+
+There are three ways you can receive commands from Direct Line and handled by your page.
```

Comment:
Commands from direct line or from a bot?

"and handled by your page" -> and handle them with your javascript? What are these samples showing - They seem to show how you add the Web Chat control to a page.